colibre/scripts/stellar_abundances_FeH_CFe_APOGEE.py

```diff
@@ -0,0 +1,157 @@
+"""
+Plots the stellar abundances ([Fe/H] vs [C/Fe]) for a given snapshot
+"""
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as pe
+import matplotlib.colors as mc
+import numpy as np
+import unyt
+import glob
+from swiftsimio import load
+from swiftpipeline.argumentparser import ScriptArgumentParser
+from velociraptor.observations import load_observations
+from unyt import unyt_array
+
+
+def read_data(data):
+    """
+    Grabs the data
+    """
+
+    mH_in_cgs = unyt.mh
+    mFe_in_cgs = 55.845 * unyt.mp
```

Comment:
Sorry for being so pedantic, but do you have a reference for these values?

Reply:
I do not have reference for those. Regarding Buder21_data.hdf5, I did not compile that dataset, James Trayford did. So let's leave it in StellarAbundances until James can add the Buder21 data treatment in the raw/ folder.